CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.25)
+cmake_minimum_required(VERSION 3.22)
```

Comment:
@ivan-cukic You committed the change to 3.25? Do you remember why? (I didn't see anything in the commit message.)

Reply:
Can't remember. If something breaks after the downgrade, we can bump the version again.